internal/http/middleware.go

```diff
@@ -0,0 +1,22 @@
+package http
+
+import "net/http"
+
+// JSON is a middleware for setting JSON http headers.
+func JSON(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		next.ServeHTTP(w, r)
+	})
+}
+
+// CORS is a middleware for setting Cross Origin Resource Sharing headers.
+func CORS(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
+		w.Header().Set("Access-Control-Allow-Methods", "POST, GET, PATCH, PUT, DELETE, OPTIONS")
```

Comment:
Any reason not to use `mux.CORSMethodMiddleware` instead?

Reply:
It's buggy rn. There is a PR to fix it but it's not been reviewed.